packages/client/tests/functional/issues/21807-citext-neon/tests.ts

```diff
@@ -0,0 +1,38 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+// Regression test for https://github.com/prisma/prisma/issues/21807
+testMatrix.setupTestSuite(
+  () => {
+    test('writing and reading a citext field works', async () => {
+      const writtenRecord = await prisma.model.create({
+        data: {
+          slug: 'someslug',
+        },
+      })
+
+      const readRecords = await prisma.model.findMany({
+        where: {
+          slug: {
+            equals: 'someslug',
```

Comment:
Could be worth changing the casing here? It should not affect the test but shows that CITEXT is working as expected?